src/linear_algebra/basic.lean

```diff
@@ -1502,4 +1514,70 @@ end
 
 end pi
 
+variables (α β)
+
+instance automorphism_group : group (β ≃ₗ[α] β) :=
+{ mul := λ f g, g.trans f,
+  one := linear_equiv.refl β,
+  inv := λ f, f.symm,
+  mul_assoc := λ f g h, by {ext, refl},
+  mul_one := λ f, by {ext, refl},
+  one_mul := λ f, by {ext, refl},
+  mul_left_inv := λ f, by {ext, exact f.left_inv x} }
+
+instance automorphism_group.to_linear_map_is_monoid_hom :
+  is_monoid_hom (linear_equiv.to_linear_map : (β ≃ₗ[α] β) → (β →ₗ[α] β)) :=
+{ map_one := rfl,
+  map_mul := λ f g, rfl }
+
+/-- The group of invertible linear maps from `β` to itself -/
+def general_linear_group := units (β →ₗ[α] β)
```

Comment:
I feel like `linear_equiv` would be a better definition, because maybe you're using things about linear equivs more often than things about units. I don't feel strongly about this.

Reply:
This definition was already there... I only moved it to below the definition of `linear_equiv`. I also don't feel strongly about this.